README.md

```diff
@@ -1,44 +1,66 @@
 # AMIBOT
 
+![Tests](https://img.shields.io/github/workflow/status/ditsuke/go-amizone/tests?label=tests&logo=github)
+[![Issues](https://img.shields.io/github/issues/asetalias/amibot?logo=github)][issues]
+![License](https://img.shields.io/github/license/asetalias/amibot)
+[<img src="https://img.shields.io/badge/-Bot-25D366?logo=Whatsapp&logoColor=fff">][whatsapp]
```

Comment:
Can we place these tech badges in a new line?

Reply:
yeah looks better with the new line